### ISSUE TITLE
Figure out how to get it to people

### ISSUE THREAD

**Issue by Lilian-Moon11** [Lilian-Moon11/medical-record-management-assistant#66]
After I have tested it a good amount reach out to have people link to it

Early testes:
Ask EDS people
Dad
Friends (Courtney for educational design)
Former coworkers (Max for accessibility), slack group projects chat

Launch:
Reach out to 
content creators (Scishow/vlog brothers, Sad Boyz, Philip Defranco)
organizations (EDS society, celiac disease foundation, dysautonomia)
give disclosure about benefit to me (possible portfolio piece), AI disclosure (considering releasing a version with no AI)

Add piece on password managers, the importance of strong passwords, and link to a password manager.
Explanation of back up key, examples of where to store like flash drive
